operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc

```diff
@@ -69,11 +69,29 @@ include::modules/olmv1-updating-an-operator.adoc[leveloffset=+1]
 * xref:../../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-about-target-versions_olmv1-installing-operator[Example custom resources (CRs) that specify a target version]
 * xref:../../operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc#olmv1-version-range-comparisons_olmv1-installing-operator[Version comparison strings]
 
-include::modules/olmv1-semver-support.adoc[leveloffset=+2]
+[id="upgrade-constraint-semantics_{context}"]
+== Upgrade constraint semantics
+
+When determining upgrade edges for an installed cluster extension, {olmv1-first} supports {olmv0-first-caps} semantics starting in {product-title} 4.16. This support matches the behavior from {olmv0}, including `replaces`, `skips`, and `skipRange` directives.
```

Comment:
There are some subtle differences, but I'm not sure if they are material, and it may not be worth calling out.

For example, consider this set of FBC channel entries:
```
- name: foo.v3.0.0
  skips: ["foo.v2.0.0"]
- name: foo.v2.0.0
  skipRange: >=1.0.0 <2.0.0
```

If `1.0.0` is installed:
- OLMv0 will not see an upgrade edge to v2.0.0 because v2.0.0 is `skip`ed and not on the replaces chain
- OLMv1 _will_ see the upgrade edge because OLMv1 does not have a concept of a replaces chain. OLMv1 just finds all entries that have a replace, skip, or skipRange that covers the currently installed version.